core/server/server_scion.go

```diff
@@ -281,13 +283,50 @@ func runSCIONServer(ctx context.Context, log *zap.Logger, mtrcs *scionServerMetr
 				continue
 			}
 
+			var NTSauthenticated bool = false
+			var ntsreq nts.NTSPacket
+			var serverCookie ntske.ServerCookie
```

Comment:
Change this to `serverCookie` in `server_ip.go` too.

Reply:
Done